docs/dotnet.rst

```diff
@@ -0,0 +1,152 @@
+
+
+6.  Create a YAML file titled
+    ``<namespace>.<app-name>.yaml`` with the following
+    example template, replacing the placeholders with the appropriate
+    information: \
+
+  .. code-block:: yaml
+
+      app-id: <namespace>.<app-name>
+      runtime: org.freedesktop.Platform
+      runtime-version: '22.08'
+      sdk: org.freedesktop.Sdk
+      sdk-extensions:
+        - org.freedesktop.Sdk.Extension.dotnet6
+      build-options:
+        prepend-path: "/usr/lib/sdk/dotnet6/bin"
+        append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+        prepend-pkg-config-path: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+
+      command: <project-name>
+
+      finish-args:
+        - --device=dri
+        # TODO: Replace this with wayland and fallback-x11 once Wayland support
+        #       becomes available:
+        #       https://github.com/AvaloniaUI/Avalonia/pull/8003
+        - --socket=x11
+        - --share=ipc
+        - --env=DOTNET_ROOT=/app/lib/dotnet
+
+      modules:
+        - name: dotnet
+          buildsystem: simple
+          build-commands:
+          - /usr/lib/sdk/dotnet6/bin/install.sh
+
+        - name: <app-name>
+          buildsystem: simple
+          sources:
+            - type: git
+              url: https://github.com/<username>/<project-name>.git
+              tag: <release-number>
+            - ./nuget-sources.json
+          build-commands:
+            - dotnet publish <project-name>/<project-name>.csproj -c Release --no-self-contained --source ./nuget-sources
+            - mkdir -p ${FLATPAK_DEST}/bin
+            - cp -r <project-name>/bin/Release/net6.0/publish/* ${FLATPAK_DEST}/bin
+
+  .. note::
+
+      For providing access to other things such as the network or
+      filesystem, see the `“Sandbox Permissions” section <https://docs.flatpak.org/en/latest/sandbox-permissions.html>`__
+
+7.  Copy and save the dotnet NuGet sources generator script
+    ``flatpak-dotnet-generator.py`` from the `Flatpak Builder Tools
+    repository <https://github.com/flatpak/flatpak-builder-tools>`__, to
+    the current folder, or run the following command to download it:
+
+  .. code-block:: shell
+
+        wget https://raw.githubusercontent.com/flatpak/flatpak-builder-tools/master/dotnet/flatpak-dotnet-generator.py
+
+8.  Clone down your project repository to the folder
+
+  .. code-block:: shell
+
+        git clone https://github.com/<username>/<app-name>.git
+
+9.  Run the NuGet source config generator script ``flatpak-dotnet-generator.py`` with the following arguments:
+
+  .. code-block:: shell
+
+        python3 flatpak-dotnet-generator.py --dotnet 6 nuget-sources.json <app-name>/<project-name>/<project-name>.csproj
+
+10. Run the Flatpak Builder script to build the local Flatpak
+
+  .. code-block:: shell
+
+        flatpak-builder build-dir <namespace>.<app-name>.yaml --force-clean
```

Comment:
```suggestion
        flatpak-builder build-dir --install-deps-from=flathub --user --force-clean --install --repo=repo <manifest>
```

This will build, install and export in one command. You can drop the other ones below.

Reply:
Just added! Thanks for pointing that out.